app/src/main/java/com/sun/noteapp/ui/trash/TrashAdapter.kt

```diff
@@ -0,0 +1,133 @@
+package com.sun.noteapp.ui.trash
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.core.content.ContextCompat
+import com.sun.noteapp.R
+import com.sun.noteapp.data.model.Note
+import com.sun.noteapp.ui.base.BaseRecyclerViewAdapter
+import com.sun.noteapp.ui.base.BaseViewHolder
+import com.sun.noteapp.utils.ColorPicker
+import com.sun.noteapp.utils.WHITE_COLOR
+import kotlinx.android.synthetic.main.item_note_trash.view.*
+
+class TrashAdapter(
+    private val listener: HandleItemClick
+) : BaseRecyclerViewAdapter<Note, TrashAdapter.ViewHolder>() {
+
+    private var selectedItems = mutableListOf<Int>()
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): ViewHolder =
+        ViewHolder(
+            LayoutInflater.from(parent.context).inflate(
+                R.layout.item_note_trash,
+                parent,
+                false
+            ),
+            listener
+        )
+
+    override fun onBindViewHolder(holder: ViewHolder, position: Int) {
+        holder.onBind(getItem(position), selectedItems)
+    }
+
+    class ViewHolder(
+        itemView: View,
+        private val listener: HandleItemClick
+    ) : BaseViewHolder<Note>(itemView),
+        View.OnClickListener,
+        View.OnLongClickListener {
+
+        private var color = WHITE_COLOR
+        private var selectedItemsTemp = mutableListOf<Int>()
+        private var noteTemp: Note? = null
+
+        fun onBind(note: Note, selectedItems: List<Int>) {
+            color = note.color
+            itemView.apply {
+                textTitleItem.text = note.title
+                textDeleteTime.text = note.hide
+                noteTemp = note
+                selectedItemsTemp.clear()
+                selectedItemsTemp.addAll(selectedItems)
+                cardViewItem.setCardBackgroundColor(
+                    ContextCompat.getColor(context, ColorPicker.getMediumColor(note.color))
+                )
+                if (selectedItems.contains(adapterPosition))
+                    setBackgroundColor(true, context)
+                else
+                    setBackgroundColor(false, context)
+                setOnClickListener(this@ViewHolder)
+                setOnLongClickListener(this@ViewHolder)
+            }
+        }
+
+        private fun setBackgroundColor(isSelected: Boolean, context: Context) {
```

Comment:
Ưu tiên context truyền ở đầu tiênhttps://github.com/framgia/coding-standards/blob/master/vn/android/codingstyleguide.md#29-th%E1%BB%A9-t%E1%BB%B1-tham-s%E1%BB%91-trong-c%C3%A1c-ph%C6%B0%C6%A1ng-th%E1%BB%A9c
```suggestion
        private fun setBackgroundColor(context: Context, isSelected: Boolean) {
```